cms/management/djangocms.py

```diff
@@ -0,0 +1,32 @@
+import os
+import sys
+
+from django.core.checks.security.base import SECRET_KEY_INSECURE_PREFIX
+from django.core.management import load_command_class
+from django.core.management.utils import get_random_secret_key
+
+
+class CMSCommandLineUtility:
+    commands = {"startproject": "cms"}
```

Comment:
Can we get a docstring here to explain what this is for. It's a bit easier than reading what it does.